src/upcoming-components/TextField/TextField.module.css

```diff
@@ -16,6 +16,17 @@
   }
 }
 
+.text-field--inline {
```

Comment:
Cases where the text is long and/or the screen size is tiny cause this to break. Made for the purposes of usage in `CADStep1`. Not sure if creating and owning a pressure tested `TextField` component with design is possible within the scope of CADStep1?

Reply:
I think this will work perfectly fine once we get rid of all of the padding in the utility class.

Reply:
Got rid of the padding and here's how it looks with 320px (which I think is our smallest view width) [Skip to content](https://github.com/chanzuckerberg/edu-design-system/pull/1049#start-of-content)
![image](https://user-images.githubusercontent.com/86632227/167958104-d7a165fc-84ca-43ae-a819-9acf5d89930a.png)

Reply:
After our discussion about legacy `InputField` and using the existing designs, removed the inline variant so now it looks like:
![Screen Shot 2022-05-12 at 10 51 59 AM](https://user-images.githubusercontent.com/86632227/168138159-72e20e2e-ec93-4208-bfeb-1b1089ae37c0.png)
![Screen Shot 2022-05-12 at 10 52 12 AM](https://user-images.githubusercontent.com/86632227/168138191-99a09ee2-69bf-44bd-9f37-5717905c9e27.png)